manifests/chain.pp

```diff
@@ -1,23 +1,43 @@
 # defined resource which creates all rules for one chain
```

Comment:
Please add some examples here. the other files contain some that you can copy. afterwards please run the rale task to regenerate the REFERENCE.md

Reply:
done